src/server/snapshot.cc

```diff
@@ -126,6 +126,9 @@ void SliceSnapshot::SerializeEntriesFb() {
   mu_.lock();
   mu_.unlock();
 
+  CHECK(!rdb_serializer_->SendFullSyncCut());
```

Comment:
Why check?

Reply:
Because I have no error propagation... I can just remove it